RELEASE_NOTES.md

```diff
@@ -1,9 +1,10 @@
-# French version
+# French
 English version below
 
 ## Fonctionnement
 
 - Amélioration significative des performances (2.6.0)
+- Pagination de l'onglet Joueur·euses (2.6.0)
```

Comment:
Do we really need to say that?
It's under the "improved performance" point (in FR and EN)